src/app/static/src/app/components/figures/strategise/strategiesTable.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <b-table :items="items"></b-table>
+</template>
+
+<script lang="ts">
+import {BTable} from "bootstrap-vue";
+import Vue from "vue";
+import {StrategyWithThreshold} from "../../../models/project";
+
+const names: Record<string, string> = {
+  "irs": "IRS only",
+  "llin-pbo": "Pyrethroid-PBO ITN only",
+  "irs-llin-pbo": "Pyrethroid-PBO ITN with IRS*",
```

Comment:
It seems a bit odd to have these values asterisked on a page which doesn't include the help text for them. Is there a plan to add that into the Strategize page?

Reply:
Good point - will add to footer of "How to use this page", consistent with regions page

Reply:
f60b175